docs/bonsai-ui/checkout/index.mdx

```diff
@@ -306,6 +306,7 @@ type Address = {
   address1: string;
   address2?: string;
   city: string;
+  province: string;
```

Comment:
Only adding here and not to `ShippingContext` since the deprecated callback is not and will not be updated.